src/test/java/seedu/address/logic/commands/SearchPublicAddressCommandTest.java

```diff
@@ -22,14 +21,14 @@ public class SearchPublicAddressCommandTest {
     @Test
     public void execute() {
```

Comment:
Remember to add test for happy flow (and any other potential cases) as well. Possibly check for correctly handing for:
- Whitespaces
- Case sensitivity
- Partial public address
- Etc.